phpcount.php

```diff
@@ -135,8 +132,8 @@ public static function GetHits($pageID, $unique = false)
         }
         else
         {
-            die("Missing hit count from database!");
-            return false;
+            //die("Missing hit count from database!");
+            return 0;
```

Comment:
If, in the web page, you call GetHits _before_ AddHits now, the entry in the database may not yet exist (for a new page). If there is no entry, I found it fit to return 0, as a defensive programming measure (I stumbled upon this after manually deleting db entries while testing :-)